clang/lib/StaticAnalyzer/Checkers/WebKit/PtrTypesSemantics.h

```diff
@@ -55,10 +55,18 @@ bool isCheckedPtr(const clang::CXXRecordDecl *Class);
 /// not, std::nullopt if inconclusive.
 std::optional<bool> isUncounted(const clang::QualType T);
 
+/// \returns true if \p Class is CheckedPtr capable AND not checked, false if
+/// not, std::nullopt if inconclusive.
+std::optional<bool> isUnchecked(const clang::QualType T);
```

Comment:
I can't seem to find a reference to this overload of `isUnchecked`. Do you see a use of it in future?

Reply:
`UncheckedCallArgsChecker::isUnsafeType` calls it.